src/roiextractors/testing.py

```diff
@@ -118,23 +118,23 @@ def generate_dummy_segmentation_extractor(
     num_rois : int, optional
         number of regions of interest, by default 10.
     num_frames : int, optional
-        _description_, by default 30
-    num_rows : number of frames used in the hypotethical video from which the data was extracted, optional
-        number of rows in the hypotethical video from which the data was extracted, by default 25.
+        description, by default 30.
```

Comment:
Can you add a description instead of this placeholder?

Reply:
I just corrected the `_description_`to  `description`, what do you mean? was that doing something different?